bciers/apps/administration/tests/components/operations/OperationInformationPage.test.tsx

```diff
@@ -98,9 +99,9 @@ describe("the OperationInformationPage component", () => {
     expect(
       screen.getByRole("heading", { name: "Registration Information" }),
     ).toBeVisible();
-    expect(screen.getByText(/Registration Category/i)).toBeVisible();
-    expect(screen.getByText(/Regulated Product Names/i)).toBeVisible();
```

Comment:
Only certain purposes have regulated products, and they're not applicable in this test